.rubocop.yml

```diff
@@ -65,6 +65,21 @@ Metrics/BlockLength:
   Exclude:
     - spec/**/*.rb
 
+Metrics/AbcSize:
+  Enabled: false
```

Comment:
I do think this metric is a useful indication that a piece of code can be simplified. I'd say either:
- Exclude the Vagrantfile and/or
- Increase the `Max` parameter (but I think we need a real test case to do that on)